examples/misc/packed_bubbles.py

```diff
@@ -0,0 +1,149 @@
+"""
+Create a packed bubble / non overlapping bubble chart to represent scalar data.
+In this example we plot the market share of different desktop browsers.
```

Comment:
If you are using a standard algorithm for this problem, please provide a link to a description. Otherwise, please give a brief summary of the algorithm.